### PR TITLE
Module 1

### PR DESCRIPTION
**Pull request by aqsa-saied-softwire**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/templates/index.html
Comment (JackMeadDev) [inline todo_app/templates/index.html:20]: Looks like this wording might be leftover from the workshop!

File: todo_app/templates/index.html
Comment (JackMeadDev) [inline todo_app/templates/index.html:21]: Very minor, but annotating this as `required` would provide some simple client-side validation

**Review by JackMeadDev — APPROVED** [review]
This looks great, you've got a clean effective approach which I had no issues running - I've left a couple of optional thoughts below but very happy for you to merge this when you're ready 😄

File: todo_app/templates/index.html
Comment (aqsa-saied-softwire) [reply todo_app/templates/index.html:21]: I've started module 2 as I was going on holiday before this would be merged in so I've added these changes into module 2's PR

**Pull request merged**
